test/counts.jl

```diff
@@ -90,6 +90,17 @@ pm = proportionmap(x)
 xx = repeat([6, 1, 3, 1], outer=100_000)
```

Comment:
Above, should test the general case with something like you did for `addcounts!`:
```julia
cm_missing = countmap(skipmissing(x))
@test cm_missing isa Dict{Int, Int}
@test cm_missing == cm

cm_any_itr = countmap((i for i in xx)) 
@test cm_any_itr isa Dict{Any,Int} #no knowledge about type
@test cm_any_itr == cm
```

Reply:
not sure what you mean here.

Reply:
I'm suggesting adding this code above, as currently you don't test `countmap` on iterators.

Reply:
aren't they on L95-L101?

Reply:
These only cover the radix sort method.